covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -77,53 +82,64 @@ def update_graph(graph_type_str, country_short_name):
     return app
 
 
-def _get_header_content(prediction_event: PredictionEvent, title: str):
-    # TODO: Rewrite the header to be general for both prediction pages.
-    # TODO: Description in the text of this date is wrong.
-    prediction_date_str = prediction_event.date.strftime("%Y-%m-%d")
-    prediction_link = "https://www.facebook.com/permalink.php?"\
+def _get_header_content(title: str):
+    mar30_prediction_link = "https://www.facebook.com/permalink.php?"\
         "story_fbid=10113020662000793&id=2247644"
     france_link = "https://www.reuters.com/article/us-health-coronavirus-france-toll/" \
         "french-coronavirus-cases-jump-above-chinas-after-including-nursing-home-tally-idUSKBN21L3BG"
-    next_day = prediction_event.date + timedelta(days=1)
     return [
         html.H1(children=title),
-        html.P(children=[
-            # TODO: fix the date
-            f'On {next_day}, mathematicians Katarína Boďová and Richard Kollár ',
-            html.A('made predictions about 7 countries', href=prediction_link),
-            f'. The data available up to that point (until {prediction_date_str}) is in the ',
-            html.Span('light green zone', style={'background-color': 'lightgreen'}),
-            f'. Data coming after {prediction_date_str} is in the white zone.',
-            dcc.Markdown(
-                """
-                The predicted number of active cases <em>N</em>(<em>t</em>) on day <em>t</em> is
-                calculated as follows (constants <em>A</em> and <em>T</em><sub><em>G</em></sub>
-                are country-specific):
-                <em>N</em>(<em>t</em>) = (<em>A</em>/<em>T</em><sub><em>G</em></sub>) ⋅
-                (<em>t</em>/<em>T</em><sub><em>G</em></sub>)<sup>6.23</sup> /
-                e<sup><em>t</em>/<em>T</em><sub><em>G</em></sub></sup>
-                """,
-                dangerously_allow_html=True,
-            ),
-        ]),
-        dcc.Markdown("""
+        dcc.Markdown(
+            f"""
+            Mathematicians Katarína Boďová and Richard Kollár predicted in March and April 2020
+            the growth of active cases during COVID-19 pandemic. Assuming social distancing measures
+            limit the rate of spread of the disease, their model suggests a polynomial growth with exponential
+            decay given by:
+
+            * <em>N</em>(<em>t</em>) = (<em>A</em>/<em>T</em><sub><em>G</em></sub>) ⋅
+              (<em>t</em>/<em>T</em><sub><em>G</em></sub>)<sup>α</sup> /
+              e<sup><em>t</em>/<em>T</em><sub><em>G</em></sub></sup>
+
+            Where:
+
+            * <em>t</em> is time in days counted from a country-specific "day one"
+            * <em>N(t)</em> is the number of active cases (cumulative positively tested minus recovered and deceased)
+            * <em>A</em>, <em>T<sub>G</sub></em> and <em>α</em> are country-specific parameters
```

Comment:
I've left the formula as HTML, since I got it for free from pandoc from my TeX formula. However, in the text, we can use markdown's stars to mark italics.
```suggestion
            * *t* is time in days counted from a country-specific "day one"
            * *N(t)* the number of active cases (cumulative positively tested minus recovered and deceased)
            * *A*, *T<sub>G</sub>* and *α* are country-specific parameters
```